frontend/src/common/utils/nameFormat.ts

```diff
@@ -0,0 +1,11 @@
+export const isNameValid = (name: string) : boolean => {
+    name = name.toLowerCase();
+    for (let i = 0; i < name.length; ++i) {
+        const c = name.charAt(i);
+        /* Checks if a character is a letter [a-z] or [A - Z] */

```

Comment:
thanks for including this. so all first names and last names must be letters only. no numbers anywhere. that we can work with